README.md

```diff
@@ -28,3 +28,11 @@ The tests implemented so far are integration (not unit) tests, so a working WAR
 3. Run `mvn jetty:run` from the eva-server subfolder
 4. Run `mvn test` from the root folder
 
+Enabling Oauth2 Security
+------------------------
+
+In order to enable Oauth2 Security you must enable the `oauth2-security` profile in the spring-boot application in any
+way supported by Spring-boot as stated in [Spring documentation](http://docs.spring.io/spring-boot/docs/current/reference/html/howto-properties-and-configuration.html#howto-set-active-spring-profiles). In adition, the application requires configuration to point to the public user check endpoints
```

Comment:
Please let the lines go as long as they can instead of adding arbitrary line breaks.